data_science/datasciencetopics.md

```diff
@@ -0,0 +1,65 @@
+# Data Science Topics and Subtopics
+
+### Terminal
+- Directories
+- Terminal commands
+- Installation and setup
+- Jupyter notebook
+
+### Git and GitHub
+- Cloning Repositories
+- Version Control
+- Branching, pull requests, working with a team
+
+### Markdown
+- Intro to basic syntax (i.e. headings, lists, code blocks, embedding links and images)
+
+### Progrmming Concepts
+- Iteration
+- Conditional Statements
+- Sequences (Array, List)
+- Data Types (Strings, numbers, logical)
+
+### Python 1 - Intro to Python
+- Integrating programming concepts
+- Dictionaries
+- Lists
+- Tables
+- Operational and Logical Operands
+
+### Python 2 - Functions
+- Writing functions
+- Try-catch

```

Comment:
```suggestion
- Exception Handling

```
Python exception handling uses `try-except(*)-else-finally`, I think it would be better to just call it Exception Handling (Not error handling, python uses the term exceptions, not errors)